README.md

```diff
@@ -10,14 +10,14 @@ CoreNEURON is a compute engine for the [NEURON](https://www.neuron.yale.edu/neur
 CoreNEURON is designed as a library within the NEURON simulator and can transparently handle all spiking network simulations including gap junction coupling with the **fixed time step method**. In order to run a NEURON model with CoreNEURON:
 
 * MOD files should be THREADSAFE
-* MOD files must use the Random123 random number generator (instead of MCellRan4)
+* If random number generator is used then Random123 should be used instead of MCellRan4
 * POINTER variables need to be converted to BBCOREPOINTER ([details here](http://bluebrain.github.io/CoreNeuron/index.html))
 
 ## Dependencies
 * [CMake 3.7+](https://cmake.org)
-* [MPI 2.0+](http://mpich.org) [Optional]
-* [PGI OpenACC Compiler >=19.0](https://www.pgroup.com/resources/accel.htm) [Optional, for GPU support]
-* [CUDA Toolkit >=6.0](https://developer.nvidia.com/cuda-toolkit-60) [Optional, for GPU support]
+* [MPI 2.0+](http://mpich.org) [Optional, for MPI support]
```

Comment:
or openmpi.org? <that is generally more consistent with python (mpi4pi)?>

Reply:
As any MPI library works, I removed link to mpich.org.